shell/components/xen/xen-state.js

```diff
@@ -51,8 +51,8 @@ let StaterMixin = Base => class extends Base {
   }
   _async(fn) {
     // TODO(sjmiles): SystemJS throws unless `Promise` is `window.Promise`
-    //return Promise.resolve().then(fn.bind(this));
-    return setTimeout(fn.bind(this), 10);
+    return Promise.resolve().then(fn.bind(this));
```

Comment:
I mentioned this to @mmandlis today but I didn't have full context -- is there an issue that would be useful to file to track whatever ramification this has for runtime logic?

Reply:
Perhaps, I wanted to see the failure one more time before posting an issue. Also, I'm generally concerned about spending time on things that aren't going to matter in the fullness of time (iow, it's likely there are bigger fish to fry).